OxfordBodleian/Ullendorff/BDLeu7.xml

```diff
@@ -34,59 +37,210 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                ልዑል፡ ኢያስ<hi rend="rubric">ተዋድዩ፡ ሰብአ፡ በነገረ፡ ኃጕል፡ እስመ፡ አንተ፡ መልአከ፡ ሣህል፡ ሰላም፡ ለከ፡</hi>                                
+                            </incipit>
+                            <explicit xml:lang="gez">
+                            በማኅፀነ፡ ሐና፡ እምኪ፡ ተማኅፀነ፡ ፈንዊዮ፡ ለፋ<supplied reason="omitted">ኑ</supplied>ኤል፡ ይዕቀብ፡ ኪያነ፡ በረምሐ፡ መስቀሉ፡ ረጊዞ፡
+                                ሰይጣነ፡ ኦአምለከ፡ ፋኑኤል፡ ዕቀባ፡ ወአድኅና፡ እምሕማመ፡ ዛር፡ ወትግሪዳ፡ ለአመትከ፡ <hi rend="rubric">ወለተ፡ ኪዳን።</hi> 
+                            </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="1r39" to="1r83"/>
+                            <title type="complete" ref="LIT6851Prayer"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">ጸሎት፡ በእንተ፡ ሕማመ፡ ቡዳ፡ ወቁመኛ፡ በስሙ፡ ለእግዚአብሔር፡ አብ፡ በስሙ፡ ለእግዚአብሔር፡ ወልድ፡ በስሙ፡ 
+                                    ለእግዚአብሔር፡ መንፈስ፡ ቅዱስ፡ ታኦስ፡ አዝ</hi>ዮስ፡ ማሲ፡ ማስያስ፡ አብፌድርስ፡ ሩፋ<gap reason="illegible" unit="chars" quantity="1"/>ን፡
+                                በጠጄን፡ ዘሐጄን፡ ዋኅ፡ ዋኅ፡ ዋኅ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                ዓምደ፡ ብርሃን፡ ልብሰ፡ ብርሃን፡ ይበርቅ፡ ቅድመ፡ ገጹ፡ ለኃይለ፡ ፀላኢ፡ ወፀር፡ ባርያ፡ ፓንዋማ፡ ፒልሜ፡ ጴልሜ፡ ኢይርአይ፡ ገጻ፡ አርእየኒ፡ ኃይለ፡ 
+                                ግርማከ፡ ንጉሠ፡ ስብሐት፡ ራማ፡ ኢይርአይ፡ ገጻከ፡ ኢይልክፍ፡ ነፍሳ፡ ወሥጋሃ፡ ለአመትከ፡ <hi rend="rubric">ወለተ፡ ኪዳን።</hi> 
+                            </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="1r83" to="1r115"/>
+                            <title type="complete" ref="LIT6852Prayer"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላክ፡ ጸሎት፡ በእንተ፡ ማእሠሮሙ፡ ለአጋንንት፡ ዘተናገሮ፡ እስክንድር፡ ንጉሥ፡ 
+                                    በዕለተ፡ ዓርብ፡ በቅድመ፡ ጎግ፡ ወማጎግ፡ በጊዜ፡ ምጽአቱ፡ ለሰ<add place="interlinear">በ</add>ድአት፡</hi> እንዘ፡ ይብል፡ አሌፍ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                ወቡዳ፡ ተያያዥ፡ ወእጀ፡ ሰብአ፡ ጽላ፡ ወጊ፡ ወፍጌን፡ ወንለ<gap reason="illegible" unit="chars" quantity="1"/>ሽ፡ ዓይነት፡ ወዓይነ፡ ወዓይነ፡ ወርቅ፡ 
+                                መንሶ፡ ወሥራይ፡ ይትሐተሙ፡ በማኅተመ፡ እሌኒ፡ ንግሥት፡ ወበማኅተመ፡ ሰሎሞን፡ ንጉሥ፡ ከመ፡ ኢይቅረቡ፡ ኀበ፡ ነፍሳ፡ ወሥጋሃ፡ 
+                                ለአመተ፡ እግዚአብሔር፡ <hi rend="rubric">ወለተ፡ ኪዳን።</hi> 
+                            </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus from="1r115" to="1r124"/>
+                            <title type="incomplete">Excerpt from <title ref="LIT5441ApostolicCreed">the Creed of the Apostles</title></title>
```

Comment:
I'm not sure, but I think that markup inside titles leads to errors. Maybe better to encode as `<title type="incomplete" ref="LIT5441ApostolicCreed">Excerpt from the Creed...`